.azure-pipelines/autorest_checks.yml

```diff
@@ -0,0 +1,47 @@
+trigger:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/core/
+
+variables:
+  NodeVersion: '10.x'
+  PythonVersion: '3.6'
+  auto_rest_clone_url: 'https://github.com/Azure/autorest.python.git'
+  repo_branch: 'azure-core'
+  source_path: 'sdk/core/azure-core'
+
+jobs:
+  - job: 'Run_AutoRest'
+    displayName: 'Run AutoRest'
```

Comment:
I didn't know you could do `displayName` for _jobs_! Definitely will be taking advantage of this.